db/model/profile.js

```diff
@@ -142,7 +142,7 @@ module.exports = function profile(seq, dataTypes) {
       }, // isAdmin
 
       hasWriteAccess(profileId, model) {
-        const accessModel = model.getAccessField();
+        const accessModel = model.getProfileAccessField();
```

Comment:
``` === 'rw'.toLowerCase()```

This doesn't make sense--we're lowercasing the string "rw"? Is it the *other* side of the equation which is supposed to be lowercased?